scripts/verify-gopkg.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash -e
```

Comment:
Did you notice anything that fails this check so far? Or is this an assert style sanity check?

This also doesn't seem to assert that the assets in the vendor/ are valid corresponding to either the toml/lock. Is that worth doing as well?

Reply:
Assert style sanity check.